app-misc/flirc/flirc-3.22.2.ebuild

```diff
@@ -35,13 +35,11 @@ RDEPEND="virtual/libusb:1
 QA_PREBUILT="/usr/bin/*"
 
 src_install () {
-	udev_newrules etc/udev/rules.d/99-flirc.rules 51-flirc.rules
```

Comment:
I'm confused. Doesn't upstream provide them anymore? FWICS that `apt.flirc.tv` thing is suited for Debian, so they should have complete Debian packages somewhere. Maybe figure out how apt gets data from it, so you could find them.